roles/kubernetes/master/handlers/main.yml

```diff
@@ -111,8 +113,6 @@
   uri:
     url: "{{ kube_apiserver_endpoint }}/healthz"
     validate_certs: no
-    client_cert: "{{ kube_apiserver_client_cert }}"
-    client_key: "{{ kube_apiserver_client_key }}"
   register: result
```

Comment:
why removing the certs here?

Reply:
I can keep them but as the validate certs is set to 'no' they are unused